girder/constants.py

```diff
@@ -195,6 +195,12 @@ class TokenScope:
     ANONYMOUS_SESSION = 'core.anonymous_session'
     USER_AUTH = 'core.user_auth'
     TEMPORARY_USER_AUTH = 'core.user_auth.temporary'
+    PLUGINS_ENABLED_READ = 'core.plugins.read'
```

Comment:
I say it's not so bad because this happens all the time with mobile apps, you'll notice they frequently change what scopes they require, and you simply have to reauthorize them whenever that happens.